common/utilities/integration_adaptors_logger.py

```diff
@@ -102,6 +102,15 @@ def _format_values_in_map(self, dict_values: dict) -> dict:
         Replaces the values in the map with key=value so that the key in a string can be replaced with the correct
         log format, also surrounds the value with quotes if it contains spaces and removes spaces from the key
         """
+
+        for key in dict_values.keys():
+            if str(key).lower() == "exception":
+                exc_inf = sys.exc_info()
+                if all(exc_inf):
+                    (etype, value, trace) = exc_inf
+                    dict_values[key] = f'{value}{traceback.format_exception(etype, value, trace)}'
+                    break
```

Comment:
Optional: I'd maybe consider moving some of this to an aptly named method that returns the formatted string, it's doesn't feel immediately obvious that this is actually retrieving the last exception from the stack - or maybe just add a line to the docstring explaining whats happening to exceptions.